Dockerfile

```diff
@@ -1,57 +1,50 @@
-FROM node:18 AS builder
-
-RUN apt-get update && apt-get install -y \
-    php \
-    php-cli \
-    php-zip \
-    php-xml \
-    php-mbstring \
-    php-curl \
-    php-sqlite3 \
-    php-ldap \
-    unzip \
-    cron
-
-RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
-
+####################
+# Build image
+####################
+FROM node:23-alpine AS builder
```

Comment:
Node.js v23 is a short-term version and will be EOL after June this year.
https://github.com/nodejs/release?tab=readme-ov-file#release-schedule

Using an even-numbered Node.js version (e.g. v22) would allow us to receive updates for a longer period (until 2027-04-30).

Reply:
Fixed